test/cql-pytest/test_cast_data.py

```diff
@@ -100,6 +100,21 @@ def test_cast_from_large_varint_to_varchar(cql, table1, cassandra_bug):
     cql.execute(f'INSERT INTO {table1} (p, cVarint) VALUES ({p}, {v})')
     assert [(str(v),)] == list(cql.execute(f"SELECT CAST(cVarint AS varchar) FROM {table1} WHERE p={p}"))
 
+# In test_cast_column_names we checked that columns are named in Cassandra style
```

Comment:
It's confusing to mention a test's name and speak in past tense ("checked") about the test that **follows**. Note that other tests in this file we use this "In ... we checked" wording, but it refers to a **previous** test, and explains that this test is different in some way.
Also since this is a test that reproduces an issue, it's important to mention it (so if we ever see this test failing, we know which issue we forgot to backport, or regressed, etc.).
How about:
```python
# Test that when SELECTing a CAST, the column names returned have the expected
# format - "cast(x as int)", as Cassandra does. Reproduces issue #14508.
```